src/graphnet/data/extractors/i3splinempeextractor.py

```diff
@@ -28,3 +28,25 @@ def __call__(self, frame: "icetray.I3Frame") -> Dict[str, float]:
             )
 
         return output
+
+
+class I3ComparisonExtractor(I3Extractor):
```

Comment:
Consider moving to its own file, e.g.,  `src/data/extractors/i3comparisonextractor.py`.

Reply:
done